includes/cleanup.php

```diff
@@ -31,19 +43,43 @@ function pmpro_delete_user_form_notice( $current_user, $userids ) {
 		}
 	}
 
+	$member_history = $wpdb->get_var( "SELECT COUNT(*) as members FROM $wpdb->pmpro_memberships_users WHERE user_id IN (" . implode( "," , $userids ) . ")" );
+
+	// Make sure that there is actually PMPro content to delete for these users.
+	if ( empty( $userids_have_levels ) && empty( $member_history ) ) {
+		// No PMPro content to delete, so we don't need to add anything to the form.
+		return;
+	}
+
+	?>
+	<div class='pmpro_delete_user_actions'>
+		<p><?php esc_html_e( 'What should be done with the PMPro Membership data for these users?', 'paid-memberships-pro' ); ?></p>
+	<?php
 	// Show a notice if users for deletion have an an active membership level.
 	if ( ! empty( $userids_have_levels ) ) { ?>
 		<div class="notice notice-error inline">
 			<?php
 			if ( count( $userids ) > 1 ) {
-				_e( '<p><strong>Warning:</strong> One or more users for deletion have an active membership level. Deleting a user will also cancel their membership and recurring subscription.</p>', 'paid-memberships-pro' );
+				echo  '<p><strong>' . esc_html__( 'Warning', 'paid-memberships-pro' ) . ': </strong>' . esc_html__( 'One or more users for deletion have an active membership level.', 'paid-memberships-pro' ) . '</p>';
```

Comment:
Need to put the : inside the translated string so RTL translations move it correctly.

Reply:
More complicated then because it's 2 translated strings and the second needs to be moved with the translation too.

So to keep the strong tags in there use wp_kses with strong as an allowed html. Then something like:

`'<p>' . wp_kses( __( '<strong>Warning:</string> One or more users for deletion have an active membership level.', 'paid-memberships-pro' ), $allowed_html ) . '</p>' );`